ffi/src/sspi/sec_winnt_auth_identity.rs

```diff
@@ -478,13 +478,18 @@ pub unsafe fn auth_data_to_identity_buffers_w(
             let auth_data = unsafe { auth_data.as_ref() }.expect("auth_data pointer should not be null");
 
             if !auth_data.package_list.is_null() && auth_data.package_list_length > 0 {
-                // SAFETY: `package_list` is not null due to a prior check.
-                *package_list = Some(String::from_utf16_lossy(unsafe {
-                    from_raw_parts(
-                        auth_data.package_list,
-                        usize::try_from(auth_data.package_list_length).unwrap(),
+                *package_list = Some(
+                    String::from_utf16(
+                        // SAFETY: `package_list` is not null due to a prior check.
+                        unsafe {
+                            from_raw_parts(
+                                auth_data.package_list,
+                                usize::try_from(auth_data.package_list_length).unwrap(),
+                            )
+                        },
                     )
-                }));
```

Comment:
nitpick: Thats’ a lot of nested items. We could consider breaking down things a little bit.